server/test_common/commonServerSteps.ts

```diff
@@ -67,7 +69,22 @@ And(
   stepWhichUpdatesWorld((world) => {
     return {
       ...world,
-      server: request(returnExpress('test-server', () => world.configuration)),
+      server: request(
+        returnExpress(
+          world.serverName || 'test-server',
```

Comment:
WIth the removal of the name argument this will need to be updated

Reply:
Just spotted it :'(